### PR TITLE
Updates installation instructions to support ARM64

### LINKED ISSUE #5164: Cannot install anaconda on Mac Mini with M2 chip
**ISIS version(s) affected**: All

**Description**  
I just purchased a new Mac Mini (2023) with an M2 chip installed. When I try to install anaconda, it says "This package is incompatible with this version of macOS. The package is trying to install content to the system volume. Contact the software manufacturer for assistance." I got the same error message for both the "regular" Mac install and the "M1" Mac install.

Do you know of a way around this? Alternatively, is there a way to install ISIS without using anaconda?

Thanks, Catherine

### PR DESCRIPTION
**Pull request by nocko**
Resolves DOI-USGS/ISIS3#5164

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by nocko** [comment]
It should be benign for intel macs to specify an osx-64 environment manually. It simplified the instructions by having them included even when not needed.

File: README.md [hunk 1]
```diff
@@ -51,14 +51,24 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
...
 1. If you are running Mac OS X, a pkg file (which looks similar to Anaconda3-5.3.0-MacOSX-x86\_64.pkg) will be downloaded. Double-click on the file to start the installation process.
 1. After the installation has finished, open up a bash prompt in your terminal window.
+1. If you have an ARM64 Mac (M1/M2) running Catalina (or later), additional prerequisites must be installed for ISIS to run in emulation:
+ - Install [XQuartz](https://www.xquartz.org/). (Tested with XQuartz 2.8.5 on MacOS Catalina)
+ - Install Rosetta2. From the terminal run: `/usr/sbin/softwareupdate --install-rosetta --agree-to-license`
 1. Next setup your Anaconda environment for ISIS. In the bash prompt, run the following commands:
 
     ```bash
+    
+    #Setup the new environment as an x86_64 environment
```
Comment (AustinSanders) [inline README.md]: Can we indicate that this command is specific to installation on mac?  This portion of the installation process was previously platform-agnostic (w.r.t. supported platforms, at least), and these commands may break a linux install.

File: README.md [hunk 2]
```diff
@@ -51,14 +51,24 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
...
+    export CONDA_SUBDIR=osx-64
+    
     #Create a new conda environment to install ISIS in
     conda create -n isis
 
     #Activate the environment
     conda activate isis
+    
+    #Force installation of x86_64 packages instead of ARM64
+    conda config --env --set subdir osx-64
```
Comment (AustinSanders) [inline README.md:72]: Same as previous comment

**Review by AustinSanders — CHANGES_REQUESTED** [review]
This PR is very much appreciated, as our development team has had no way to officially support (or test an emulated version of) ISIS on ARM.

One small change request and this will be ready for merge.

**Review by AustinSanders — APPROVED** [review]

**Pull request merged**